### PR TITLE
Tas 54 qa testing

### PR DESCRIPTION
**Pull request by sandeshkulk**
Updated Workflow, to upload apk to github, also added email and password resource tag for testing.
You can look at the Artifacts being uploaded during runtime.

### CODE REVIEW

**Review by psavarmattas — APPROVED** [review]

File: .github/workflows/android_workflow.yaml [hunk 1]
```diff
@@ -73,6 +73,12 @@ jobs:
       - name: Assemble app APK (debug)
         run: ./gradlew assembleDebug
 
+      - name: Upload APK
+        uses: actions/upload-artifact@v3
+        with:
+          name: apk
+          path: /home/runner/work/TaskMuse/TaskMuse/app/build/outputs/apk/debug/app-debug.apk
+
```
Comment (Agata-Duda) [inline .github/workflows/android_workflow.yaml:81]: Are these changes already included in @psavarmattas PR for login fields?

Comment (psavarmattas) [reply .github/workflows/android_workflow.yaml:81]: Yes they are they are authored by sandesh in my branch.

Comment (Agata-Duda) [reply .github/workflows/android_workflow.yaml:81]: So we can rebase this branch with main and I'll approve

Comment (psavarmattas) [reply .github/workflows/android_workflow.yaml:81]: We can merge it directly also it won't create a conflict

**Review by Agata-Duda — APPROVED** [review]

**Pull request merged**